src/aosm/azext_aosm/cli_handlers/onboarding_core_vnf_handler.py

```diff
@@ -82,13 +82,19 @@ def _get_processor_list(self) -> List[AzureCoreArmBuildProcessor | VHDProcessor]
         # Instantiate vhd processor
         if not self.config.vhd.artifact_name:
             self.config.vhd.artifact_name = self.config.nf_name + "-vhd"
+
+        if self.config.vhd.file_path:
+            file_path = Path(self.config.vhd.file_path).absolute()
+        else:
+            file_path = None
```

Comment:
I don't think we need this else statement, do we?

Not a big change, but two lines less is still good :-)

Reply:
We chatted. Keeping the else